<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2199</width>
    <height>1377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>630</y>
      <width>351</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Pick">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>500</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="START">
    <property name="geometry">
     <rect>
      <x>1940</x>
      <y>620</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_12">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_13">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l12">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>720</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l14">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l6">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="yourleft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l7">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_14">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_10">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_7">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="myright">
    <property name="geometry">
     <rect>
      <x>2070</x>
      <y>1090</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myleft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1090</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l13">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="yourright">
    <property name="geometry">
     <rect>
      <x>2070</x>
      <y>150</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l10">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>700</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l11">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_11">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>880</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_6">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>970</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="l8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2199</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
